share/doc/homebrew/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md

```diff
@@ -1,52 +1,70 @@
+2. Change to the directory containing Homebrew formulae with `cd $(brew --repository homebrew/core)`.
+3. Add your pushable forked repository with `git remote add <YOUR_USERNAME> https://github.com/<YOUR_USERNAME>/homebrew-core.git`
+ * `<YOUR_USERNAME>` is your GitHub username, not your local machine username.
+
+For formulae in central taps other than `homebrew/core`, such as `homebrew/science` or `homebrew/games`, substitute that tap's name for `homebrew/core` in each step, and alter the GitHub repository URLs as necessary.
+
+## Create your pull request from a new branch
+
+To make a new branch and submit it for review, create a GitHub pull request with the following steps:
+
+1. Check out the `master` branch with `git checkout master`.
+2. Retrieve new changes to the `master` branch with `brew update`.
+3. Create a new branch from the latest `master` branch with `git checkout -b <YOUR_BRANCH_NAME> origin/master`.
+4. Make your changes to any Homebrew formula with `brew edit` or your favorite text editor, following all the guidelines in the [Formula Cookbook](Formula-Cookbook.md).
+ * If there's a `bottle do` block in the formula: don't remove or change it; we'll update it when we pull your PR.
+5. Test your changes on each changed formula by doing the following, and ensure they all pass without issue. (Make sure you do the `brew audit` step while your changed formula is installed.)
+ 1. `brew tests`
+ 2. `brew install <CHANGED_FORMULA>`
+ 3. `brew test <CHANGED_FORMULA>`
+ 4. `brew audit --strict <CHANGED_FORMULA>`
+6. Make a separate commit for each changed formula with `git add` and `git commit`.
+7. Upload your new commits to the branch on your fork with `git push --set-upstream <YOUR_USERNAME> <YOUR_BRANCH_NAME>`.
+8. Go to https://github.com/Homebrew/homebrew-core and create a pull request to request review and merging of the commits in your pushed branch. Explain why the change is needed and, if fixing a bug, how to reproduce the bug. Make sure you have done each step in the checklist that appears in your new PR.
+ * Please note that our preferred commit message format for simple version updates is "`<FORMULA_NAME> <NEW_VERSION>`", e.g. "`source-highlight 3.1.8`". `devel` version updates should have the commit message suffixed with `(devel)`, e.g. "`nginx 1.9.1 (devel)`". If updating both stable and `devel`, just list the stable (main) version.
+9. Await feedback or a merge from Homebrew's maintainers. We typically respond to all PRs within a couple days, but it may take up to a week, depending on the maintainers' workload.
+10. Thank you!
 
 ## Following up
 
 To respond well to feedback:
 
-1. Ask for clarification of anything you don't understand and help with anything you don't know how to do.
+1. Ask for clarification of anything you don't understand, and for help with anything you don't know how to do.
```

Comment:
It is. This is not an "Oxford comma". I'm not sure what you'd call this comma, but I think it's correct (though not mandatory). Maybe I'm just misapplying the rule about independent clauses here, though. (Since the stuff after the comma isn't an independent clauses.)